test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
+    Given Minishift has state "Running"
+     When executing "oc login --username=developer --password=developer" retrying 24 times with wait period of 5 seconds
```

Comment:
This conception is inherited from 2.* downstream releases [https://github.com/projectatomic/adb-tests/blob/master/cdk-testsuite/utils/openshiftUtils.py](url). In 2.* after ```vagranrt up``` we had timing issue  with immediate ```oc login``` which was resolved as per the above implementation. I believe this is not the same case here. In the above step i removed the retry and it went fine without error/exception.